apps/web/src/routes/AuthLayoutRouter/components/GlobalNav/GlobalNavItem.jsx

```diff
@@ -54,8 +146,45 @@ export default function GlobalNavItem ({
             {badgeCount > 0 && <Badge number={badgeCount} className='absolute -top-2 -left-2' expanded />}
           </div>
         </TooltipTrigger>
-        {tooltip && <TooltipContent side='right' arrow>{tooltip}</TooltipContent>}
+        {tooltip && (
+          <TooltipContent 
+            side='right' 
+            className={cn(
+              'transition-all duration-300 ease-out transform',
+              {
+                'opacity-60 translate-x-0 scale-80': parentShowTooltip && !isHovered && shouldAnimate,
+                'opacity-100 translate-x-0 scale-110': isHovered,
+                'opacity-0 -translate-x-2': !shouldAnimate
+              }
+            )}
+            style={{
+              // Only apply transition delay for parent hover cascade
+              transitionDelay: !isHovered && parentShowTooltip ? `${index * 100}ms` : '0ms',
+              // Ensure tooltip doesn't appear too low
+              maxHeight: 'calc(85vh)',
+              bottom: 'auto'
+            }}
+          >
+            {tooltip}
+          </TooltipContent>
+        )}
       </div>
     </Tooltip>
   )
 }
+
+// Add this component to your file
+export function GlobalNavTooltipContainer({ children }) {
```

Comment:
unfortunately switching to Vite means that we can only export 1 component per file for hot reloading to work. so this needs to be moved into into its own file. can be in the same GlobalNav directory

Reply:
This was leftover from an attempt to make the tooltips scroll more smoothly -- I'm removing it.